mate-panel/panel-run-dialog.c

```diff
@@ -940,6 +981,19 @@ panel_run_dialog_add_items_idle (PanelRunDialog *dialog)
 				    COLUMN_PATH,      matemenu_tree_entry_get_desktop_file_path (entry),
 				    COLUMN_VISIBLE,   TRUE,
 				    -1);
+		if (i < G_N_ELEMENTS (accelerator_key_mapping)) {
+			gtk_list_store_set(dialog->program_list_store, &iter,
+								COLUMN_ACCELERATOR_MASK, (gint)accelerator_key_mapping[i].modifier,
+								COLUMN_ACCELERATOR_KEY_VALUE, accelerator_key_mapping[i].key_id,
+								-1);
+			g_hash_table_insert (accelerator_keys_to_tree_iter_map, GUINT_TO_POINTER(accelerator_key_mapping[i].key_id), GINT_TO_POINTER(i));
+			i++;
+		} else {
+			gtk_list_store_set(dialog->program_list_store, &iter,
+								COLUMN_ACCELERATOR_MASK, (gint)GDK_MOD1_MASK,
+								COLUMN_ACCELERATOR_KEY_VALUE, 0,
+								-1);
```

Comment:
Same formatting issue here and missing blank after gtk_list_store_set.

Reply:
Fixed (fixup 6cff6f3d2e9fd695399328e5597ac7bfbad2e2ab)